cmd/minikube/cmd/status.go

```diff
@@ -490,6 +501,7 @@ func clusterState(sts []*Status) ClusterState {
 
 		Components: map[string]BaseState{
 			"kubeconfig": {Name: "kubeconfig", StatusCode: statusCode(sts[0].Kubeconfig)},
+			"timetostop": {Name: "timetostop", StatusCode: statusCode(timeToStopStatusCode)},
```

Comment:
Instead of a component, I think it would make more sense to have this higher level in `ClusterState`. That way we don't need to worry about status codes either.

Right now, it looks like this:

```json
{
  "Name": "minikube",
  "StatusCode": 200,
  "StatusName": "OK",
  "BinaryVersion": "v1.15.1",
  "Components": {
    "kubeconfig": {
      "Name": "kubeconfig",
      "StatusCode": 200,
      "StatusName": ""
    },
    "timetostop": {
      "Name": "timetostop",
      "StatusCode": 200,
      "StatusName": ""
    }
  },
  "Nodes": [
    {
      "Name": "minikube",
      "StatusCode": 200,
      "StatusName": "OK",
      "Components": {
        "apiserver": {
          "Name": "apiserver",
          "StatusCode": 200,
          "StatusName": "OK"
        },
        "kubelet": {
          "Name": "kubelet",
          "StatusCode": 200,
          "StatusName": "OK"
        }
      }
    }
  ]
}
```

and the following would be simpler:

```json
{
  "Name": "minikube",
  "StatusCode": 200,
  "StatusName": "OK",
  "BinaryVersion": "v1.15.1",
  "TimeToStop": "23s",
  "Components": {
    "kubeconfig": {
      "Name": "kubeconfig",
      "StatusCode": 200,
      "StatusName": ""
    }
  },
  "Nodes": [
    {
      "Name": "minikube",
      "StatusCode": 200,
      "StatusName": "OK",
      "Components": {
        "apiserver": {
          "Name": "apiserver",
          "StatusCode": 200,
          "StatusName": "OK"
        },
        "kubelet": {
          "Name": "kubelet",
          "StatusCode": 200,
          "StatusName": "OK"
        }
      }
    }
  ]
}
```

Reply:
done